app/models/tag.rb

```diff
@@ -94,10 +92,10 @@ def self.sorted_parents
   end
 
   def sorted_children
-    if self.child_ordering == "alphabetical"
-      children.in_alphabetical_order
+    if child_ordering == "alphabetical"
+      children.sort_by(&:title)
```

Comment:
This is not desirable - we have had examples (eg, "IR35") where the ordering was wrong due to a case sensitive sort being used.